package.json

```diff
@@ -4,8 +4,9 @@
   "description": "replacement for `npm version` with automatic CHANGELOG generation",
   "bin": "bin/cli.js",
   "scripts": {
-    "fix": "eslint . --fix",
-    "posttest": "eslint . && npm run format:check",
+    "eslint": "eslint .",
+    "eslint:fix": "npm run eslint -- --fix",
```

Comment:
Nice. Thanks for splitting this out.

What do you think of `lint` instead of `eslint`? 

I usually use the name of what it's for, rather than the name of the tool - that way you don't need to remember whether the project has different tools. What do you think?

Reply:
I've made this change on master, assuming it wouldn't be terribly contentious

Reply:
It's fine for me. There are some cases where there is more than one linter (eg Stylelint + ESLint). In that scenario, a "lint" script usually calls all the other scripts. 

In this case, your suggestion makes sense given that ESLint is the only linter in this project (omitting prettier).